graphql/util.go

```diff
@@ -1323,15 +1323,22 @@ func GroupProjects(projects []model.ProjectRef, onlyDefaultedToRepo bool) ([]*Gr
 			GroupDisplayName: groupName,
 			Projects:         groupedProjects,
 		}
-
-		if groupedProjects[0].UseRepoSettings && groupedProjects[0].RepoRefId != nil {
-			repoRef, err := model.FindOneRepoRef(utility.FromStringPtr(groupedProjects[0].RepoRefId))
+		project := groupedProjects[0]
+		if project.UseRepoSettings {
+			repoRefId := utility.FromStringPtr(project.RepoRefId)
+			repoRef, err := model.FindOneRepoRef(repoRefId)
 			if err != nil {
 				return nil, err
 			}
 
 			apiRepoRef := restModel.APIProjectRef{}
-			if repoRef != nil {
+			if repoRef == nil {
+				grip.Warning(message.WrapError(err, message.Fields{
+					"message":   "repoRef not found ",
```

Comment:
nit: extra space, and by convention we use snake case (repo_ref_id) for splunk. I think we can log this at the actual Error level also, since this does indicate a real error.